testing/test_compat.py

```diff
@@ -52,8 +51,8 @@ def __getattr__(self, attr):
     with pytest.raises(
         ValueError,
         match=(
-            "could not find real function of <Evil left=800>\n"
-            "stopped at <Evil left=800>"
+            "could not find real function of <Evil left=900>\n"
```

Comment:
explanation: previously we were doing 2 get attributes but it's now replaced by isinstance check. So left is 100 more.